### ISSUE TITLE
Optimistic updates

### ISSUE THREAD

**Issue by tappleby** [acdlite/redux-promise#12]
This is related to the `sequence` discussion in acdlite/flux-standard-action#7.

If `payload` is a Promise, where should data needed to perform the optimistic update be put? `meta`?

Wherever this data ends up being stored I think it [should become the payload](https://github.com/acdlite/redux-promise/blob/sequence/src/index.js#L21) for first action in sequence.

**Comment by acdlite** [comment]
We're using the `sequence` branch of redux-promise and redux-actions at OpenGov, and we put data needed for optimistic updates inside of meta. Not sure about making it the payload of the first action — it seems redundant, if it's already part of meta.

**Comment by tappleby** [comment]
Makes sense, my initial though was it felt weird having to look at `meta` while in the reducer instead of `payload`. After thinking about it some more, it would probably be confusing for a value you put in `meta` to move to `payload`.

**Comment by kwelch** [comment]
Is there an npm version that includes can be used to use the sequence updates prior to it being part of spec?
